### PR TITLE
Improvements to application details page

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+ - MN026
+---
+
+This iteration contains several improvements to the way we present the candidate’s application to providers.
+
+## Marking roles that are relevant to teaching
+
+We used to mark roles that involved working with children.
+
+But research shows that a role can be relevant to teaching even if it doesn’t involve working with children.
```
Comment (frankieroberto) [inline app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:17]:
```suggestion
But research shows that a job can be relevant to teaching even if it didn’t involve working with children.
```

File: app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ - MN024
+ - MN025
+ - MN026
+---
+
+This iteration contains several improvements to the way we present the candidate’s application to providers.
+
+## Marking roles that are relevant to teaching
+
+We used to mark roles that involved working with children.
```
Comment (frankieroberto) [inline app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:15]:
```suggestion
We used to mark jobs that involved working with children.
```

File: app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+date: 2021-02-04
+tags:
+ - MN024
+ - MN025
+ - MN026
+---
+
+This iteration contains several improvements to the way we present the candidate’s application to providers.
+
+## Marking roles that are relevant to teaching
```
Comment (frankieroberto) [inline app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:13]:
```suggestion
## Marking jobs that are relevant to teaching
```

File: app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+We used to mark roles that involved working with children.
+
+But research shows that a role can be relevant to teaching even if it doesn’t involve working with children.
+
+We are instead going to ask candidates whether roles are relevant to teaching, so we changed the flag which can appear against work history items.
```
Comment (frankieroberto) [inline app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:19]:
```suggestion
We are instead going to ask candidates whether jobs are relevant to teaching, so we changed the message which can appear within work history items.
```

File: app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+## Removed job descriptions
```
Comment (frankieroberto) [inline app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:21]:
```suggestion
## Removed relevant skills and experience
```

File: app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+We used to ask candidates to enter descriptions of their roles. This created more work for candidates and also for providers, who had to read descriptions which were sometimes irrelevant.
```
Comment (frankieroberto) [inline app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:23]:
```suggestion
We used to ask candidates to enter describe their role and how they developed skills and experience relevant to teaching. This created more work for candidates, particularly for old jobs not relevant to teaching, and also for providers, who had to read descriptions which were sometimes irrelevant. 
```

File: app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+Job descriptions should still be shown for old applications where the candidate did provide this information.
+
+## Making it easier to spot gaps in work
+
+Providers need to spot periods of time in which a candidate was not working or getting unpaid experience. 
+
+We have combined work experience and unpaid roles into one list, so that providers do not need to compare two separate lists. We mark unpaid experience so providers can distinguish between paid and unpaid roles.
+
+We have always indicated gaps but we now use the inset text component to make it easier to spot them.
```
Comment (frankieroberto) [inline app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:35]: > We have always indicated gaps but we now use the inset text component to make it easier to spot them.

Is this true? I thought previously only gaps that had an explanation were shown?

Comment (adamsilver) [reply app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:35]: yes it’s true

image

Comment (frankieroberto) [reply app/posts/manage-teacher-training-applications/2021-02-04-application-page-improvements.md:35]: Interesting!

**Pull request merged**